src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,45 @@
+package pages;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+
+public class HomePage extends BasePage {
+
+    public HomePage(WebDriver driver) {
+            super(driver);
+        }
+//    public HomePage(WebDriver driver) {
+//        super(driver);
+//    }
+
+        public void clickOnPlaylist(String playlistName) {
+            wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//a[contains(text(),'" + playlistName + "')]"))).click();
```

Comment:
We can make the xpath or css the class variable (Locator)

For example

![image](https://github.com/Test-Pro-Bootcamp/A49-QA-Automation-Framework/assets/104761482/d327eccd-5b31-455b-b586-1bbad7d539d5)